rhino3dm/py/SampleCreateCircleAndWriteTo3dmFile.py

```diff
@@ -0,0 +1,11 @@
+import rhino3dm
+import random
+
+model = rhino3dm.File3dm()
+for i in range(20):
+	pt = rhino3dm.Point3d(random.uniform(-10,10), random.uniform(-10,10), 0)
+	model.Objects.AddPoint(pt)
+	circle = rhino3dm.Circle(pt, random.uniform(1,4))
+	model.Objects.AddCircle(circle)
+	
+model.Write("circle.3dm", 7)
```

Comment:
sorry, I was unclear. The 7 should be removed so we don't deal with version numbers in the future.
`model.Write("circle.3dm")` should work

Reply:
Ok, You are right. I will correct.